OvmfPkg/PlatformCI/.azurepipelines/Ubuntu-GCC5.yml

```diff
@@ -19,7 +19,7 @@ jobs:
   - job: Platform_CI
     variables:
       package: 'OvmfPkg'
-      vm_image: 'ubuntu-18.04'
+      vm_image: 'ubuntu-latest'

```

Comment:
would you use "ubuntu-22.04" instead of ubuntu-latest ? latest = 20.04 now.

https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml